clients/banking/src/App.tsx

```diff
@@ -34,20 +34,9 @@ export const App = () => {
   return (
     <ErrorBoundary
       key={route?.name}
-      fallback={({ error }) => {
-        const is401 =
-          error instanceof CombinedError
-            ? error?.graphQLErrors?.some(
-                item =>
-                  (item.extensions.response as { status: number } | undefined)?.status === 401,
-              )
-            : false;
-        return is401 ? (
-          <LoadingView color={colors.gray[400]} style={styles.base} />
-        ) : (
-          <ErrorView error={error} style={styles.base} />
-        );
-      }}
+      fallback={({ error }) =>
+        isUnauthenticatedError(error) ? <></> : <ErrorView error={error} style={styles.base} />
```

Comment:
nit: will it be similar to return `null` instead of `<></>`?

Reply:
You can't, the typing expect a `ReactElement`

<img width="1548" alt="Screenshot 2023-05-30 at 15 16 05" src="https://github.com/swan-io/swan-partner-frontend/assets/1902323/29c365e2-92e2-4aee-924f-e6c7a3c71933">

Reply:
perfect thanks a lot!